src/main/kotlin/no/nav/familie/ef/iverksett/tekniskopphor/IverksettTekniskOpphørTask.kt

```diff
@@ -0,0 +1,47 @@
+package no.nav.familie.ef.iverksett.tekniskopphor
+
+import no.nav.familie.ef.iverksett.iverksetting.IverksettingRepository
+import no.nav.familie.ef.iverksett.iverksetting.tilstand.TilstandRepository
+import no.nav.familie.ef.iverksett.økonomi.OppdragClient
+import no.nav.familie.ef.iverksett.økonomi.utbetalingsoppdrag.UtbetalingsoppdragGenerator
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.domene.TaskRepository
+import org.springframework.stereotype.Service
+import java.util.UUID
+
+@Service
+@TaskStepBeskrivelse(
+        taskStepType = IverksettTekniskOpphørTask.TYPE,
+        beskrivelse = "Utfører iverksetting av teknisk opphør mot økonomi."
+)
+class IverksettTekniskOpphørTask(val iverksettingRepository: IverksettingRepository,
+                                 val oppdragClient: OppdragClient,
+                                 val taskRepository: TaskRepository,
+                                 val tilstandRepository: TilstandRepository
+) : AsyncTaskStep {
+
+    override fun doTask(task: Task) {
+        val behandlingId = UUID.fromString(task.payload)
+        val tekniskOpphør = iverksettingRepository.hentTekniskOpphør(behandlingId)
+        val forrigeTilkjentYtelse = tekniskOpphør.forrigeBehandlingId.let {
+            tilstandRepository.hentTilkjentYtelse(it) ?: error("Kunne ikke finne tilkjent ytelse for behandlingId=${it}")
+        }
+        val utbetaling =
+                UtbetalingsoppdragGenerator.lagTilkjentYtelseMedUtbetalingsoppdrag(tekniskOpphør.tilkjentYtelseMedMetaData,
```

Comment:
Tolker det som att på rad 30 i `Sekvens1.csv` så testes det att man sender inn en tom liste til lagTilkjentYtelseMedUtbetalingsoppdrag men hade kanskje vært fint med en explicit unittest for den (unittest for UtbetalingsoppdragGenerator)

Hadde nesten kunnet tenkt meg å se en integrasjonstest for denne tasken, sånn att man legger inn en behandling først, og sen kaller på doTask i denne som utfører allt arbeid. Tenker man får testet att allt henger sammen då

Reply:
lagt til test her https://github.com/navikt/familie-ef-iverksett/pull/87/commits/51dc48f3261bfbd6d42b1874e4a866be9605d3a7